membership-attribute-service/app/controllers/PaymentUpdateController.scala

```diff
@@ -169,10 +171,16 @@ class PaymentUpdateController(
             lastName = billToContact.lastName,
             countryCode = "GB",
           )
+          paymentGatewayToUse =
+            if (paymentGatewayOwner.equals("tortoise-media")) {
+              GoCardlessTortoiseMediaGateway
+            } else {
+              GoCardlessGateway
+            }
```

Comment:
I'd recommend to miss out the braces if it's only a simple if expression like that

(or you could go for a pattern match if you think it looks neater for dealing with the Option:
```suggestion
            paymentGatewayOwner match {
              case Some("tortoise-media") => GoCardlessTortoiseMediaGateway
              case _ => GoCardlessGateway
            }
```
)

Reply:
Thanks @johnduffell , since `paymentGatewayOwner` is an Option[String], I think your suggestion is better 💪